core-aam/core-aam.html

```diff
@@ -3397,12 +3392,13 @@
 		<section>
 			<h2>Substantive changes since the <a href="http://www.w3.org/TR/2015/WD-core-aam-1.1-20151119/">last public working draft</a></h2>
 			<ul>
+				<li>14-Mar-2017: Undid the 1.1 addition of RELATION_POPUP_FOR to ATK/AT-SPI mappings of aria-haspopup. While the addition is correct in terms of platform expectations, the ARIA spec provides no means through which user agents could reliably implement this.</li>
```

Comment:
@Joanmarie how about "Removed" instead of "Undid", since it's the opposite of "Added"?